go.mod

```diff
@@ -96,7 +96,7 @@ require (
 	go.uber.org/atomic v1.9.0 // indirect
 	go.uber.org/multierr v1.8.0 // indirect
 	golang.org/x/crypto v0.9.0 // indirect
-	golang.org/x/exp v0.0.0-20220303212507-bbda1eaf7a17 // indirect
+	golang.org/x/exp v0.0.0-20220303212507-bbda1eaf7a17
```

Comment:
Do you know why this changed from an indirect reference?

Reply:
I think it's because `pkg/storage/governance_request_feedback.go` uses `golang.org/x/exp/slices`.

Reply:
Thanks Dylan!